### PR TITLE
Homepage files restructure

### PR DESCRIPTION
**Pull request by kat1906**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -38,7 +38,8 @@ class App {
 
     // Nunjucks configuration
     const appViews = [
-      path.join(__dirname, 'app/views/'),
+      path.join(__dirname, 'app/homepage'),
+      path.join(__dirname, 'app/views'),
```
Comment (alibombaye) [inline app.js]: I think we can just have `app` as the base root and then each template is from this path

Comment (kat1906) [reply app.js]: I don't know what you mean

Comment (alibombaye) [reply app.js]: we can just have the one path
`path.join(__dirname, 'app'),`
I don't think we should have to add more paths for each page.

Comment (kat1906) [reply app.js]: we need them both for now until all sections are done and all shared components come from the npm modules

**Pull request merged**